include/linux/bpf.h

```diff
@@ -46,6 +46,13 @@ enum bpf_stats_type
     BPF_STATS_TYPE_UNKNOWN
 };
 
+// All types below must be ABI compatible with the Linux bpf() syscalls. This means
+// that the order, size and alignment of the types must match uapi/linux/bpf.h.
```

Comment:
... in which version of Linux?

Reply:
Any version really. How about "mainline Linux" to mean whatever is in Linus' tree?

Reply:
Added some extra prose to this effect.

Reply:
Ok.  Since the kernel version parameter is ignored I guess it doesn't matter for now.  I'm worried it could conceivable cause a problem in the future, but we can defer that until the time comes.